devicetypes/smartthings/zwave-multi-button.src/zwave-multi-button.groovy

```diff
@@ -54,7 +58,11 @@ def updated() {
 def initialize() {
 	def numberOfButtons = prodNumberOfButtons[zwaveInfo.prod]
 	sendEvent(name: "numberOfButtons", value: numberOfButtons, displayed: false)
-	sendEvent(name: "checkInterval", value: 8 * 60 * 60 + 2 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
+	if(zwaveInfo.mfr?.contains("0371")) {
+		sendEvent(name: "DeviceWatch-Enroll", value: JsonOutput.toJson([protocol: "zwave", scheme:"untracked"]), displayed: false)
+	} else {
+		sendEvent(name: "checkInterval", value: 8 * 60 * 60 + 2 * 60, displayed: false, data: [protocol: "zwave", hubHardwareId: device.hub.hardwareID])
```

Comment:
Is the wake up interval 4 hours? 

Sorry for repeating myself across multiple PRs, but the check interval needs to be 
```

func calculate_check_interval(w: wakeup_interval)
{
    check_interval = 2 * w + 10 minutes;
}
```

The 10 minutes should be the offset used going forward, the reason being that if we miss the first wake up interval (at the radio level), the device could stay up for 5 minutes before going back to sleep and wake up 4 hours or so later. Also most of these low power devices are running on Low power internal clocks which can be inaccurate, so the 4 hours wake up interval could sleep by seconds or more, so the 10 minutes should be a decent cushion that when elapsed we can feel safe to mark the device offline

Reply:
Sorry, totally forgot about it. Now it should be ok.